rollbar-cli/rollbar-cli.cabal

```diff
@@ -42,6 +42,7 @@ library
       base >=4.13 && <5
     , optparse-applicative >=0.14 && <1
     , rollbar-client >=1.0 && <2
+    , bytestring >=0.10 && <1
```

Comment:
I don't think you are making use of the dependency in the library code, so this could be removed.